pdfminer/pdffont.py

```diff
@@ -140,12 +130,18 @@ def do_keyword(self, pos, token):
 
 
 NIBBLES = ('0', '1', '2', '3', '4', '5', '6', '7', '8', '9', '.', 'e', 'e-', None, '-')
+"""
```

Comment:
I might be nitpicking here: according to pep8 block comments should start with a hashtag instead of using a multi-line string. 

Advantage of using hashtag: an intelligent editor can understand that it is a comment

Reply:
Got it, Done.